test/sse/test_sse.h

```diff
@@ -547,9 +550,10 @@ __m128 ExtractIntInRandomOrder(unsigned int *arr, int i, int n, int prime) {
         char str[256]; tostr(&m1, str); \
         char str2[256]; tostr(&ret, str2); \
         printf("%s(%s, 0x%08X, %d) = %s\n", #func, str, interesting_ints[j], Tint, str2); \
-      }
+      } \
+  }();
```

Comment:
Can we use simple C functions instead of C++ lambda's perhaps?   That would also help with debug-ability as it would yield meaningful backtraces.    If its not easy then this change is still better of course.  lgtm either way.

Reply:
I don't know how to do that easily. The function body would then need to be emitted somewhere else by one set of macro expansion magic, and then calls to those functions separately in another place.